src/hooks/useFiatText.ts

```diff
@@ -107,7 +106,7 @@ export const formatFiatString = (props: {
  * Returns a localized fiat amount string
  * */
 export const displayFiatAmount = (fiatAmount?: number | string, precision: number = 2, noGrouping: boolean = true) => {
-  const fiatAmountBns = fiatAmount != null ? toBigNumberString(fiatAmount) : undefined
+  const fiatAmountBns = fiatAmount != null ? fiatAmount : undefined
```

Comment:
This becomes a noop. Also the conditions below are defunct from the type change from string to number.. Either change the condition or you could do `const fiatAmountBns = fiatAmount != null ? fiatAmount.toString() : undefined` as a lazy fix.

Reply:
Sorry, my fixup accidentally got merged in. I replaced the call to `toBigNumberString` with a call to `add( , '0')`, which should do the same thing.